node/src/main/java/de/obsidiancloud/node/network/packets/N2NScreenPacket.java

```diff
@@ -0,0 +1,50 @@
+package de.obsidiancloud.node.network.packets;
+
+import de.obsidiancloud.common.OCPlayer;
+import de.obsidiancloud.common.ObsidianCloudAPI;
+import de.obsidiancloud.common.command.CommandExecutor;
+import de.obsidiancloud.common.network.Packet;
+import de.obsidiancloud.node.ObsidianCloudNode;
+import io.netty.buffer.ByteBuf;
+import org.jetbrains.annotations.NotNull;
+
+public class N2NScreenPacket extends Packet {
+    private String server;
+    private CommandExecutor executor;
+
+    @Override
+    public void write(@NotNull ByteBuf byteBuf) {
+        writeString(byteBuf, server);
+        byteBuf.writeBoolean(executor instanceof OCPlayer);
+        if (executor instanceof OCPlayer) {
```

Comment:
Lass den executor direkt in der instanceof casten, also

`if (executor instanceof OCPlayer player)`